shared/scripts/web-terminal.sh

```diff
@@ -60,7 +60,7 @@ http {
 EOF
 
 echo "--> Installing systemd script for Wetty web terminal"
-sudo tee /etc/systemd/system/wetty.service > /dev/null <<"SERVICE"
```

Comment:
I believe these quotes are necessary to disable parameter substitution. I believe tests are failing because of it.